.github/workflows/build.yml

```diff
@@ -139,6 +139,7 @@ jobs:
           else
             export CC='clang'
             export CXX='clang++'
+            export CLANG_VERSION='19'
```

Comment:
I don't think this is the right way to switch clang version -> there is a default of 12 still in the build script https://github.com/stellar/stellar-core/blob/f5fba53aaec6ce05c709af34af4cea1ae3997390/ci-build.sh#L80